packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -0,0 +1,129 @@
+/* global browser, Terra */
+describe('Form-Input', () => {
```

Comment:
These webdriver tests should be testing more than screenshot comparison and accessiblity. We also want to add test to ensure functionality behaves as expected. Adding functionality tests wound't necessarily require a screenshot afterwards, unless styles change for a behavior.

Reply:
@JakeLaCombe and I synced about these tests and determined that functional testing would be done via the jest tests.

Reply:
👍 Sounds good.

Reply:
Done c33141d942293cae899fc6e6efcd0587346cc7db